.builder/actions/crt_size_check.py

```diff
@@ -9,33 +9,29 @@
 class CrtSizeCheck(Builder.Action):
     def run(self, env):
         # Maximum package size in bytes
-        max_node_size = 5_000_000
+        max_size = 5_000_000
         # Maximum dist folder size in bytes
-        max_dist_size = 50_000_000
-        # Size of current folder
-        current_folder_size = 0
-        # Total size of files in dist folder
         total_size = 0
 
-        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/')):
-            current_folder_size = 0
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/native/')):
```

Comment:
I was wrong, maybe just leave `lib` and `include` out, which are C code and libraries that will not be included in the package we are shipping to npm